projects/demo/project.json

```diff
@@ -176,7 +176,7 @@
             "builder": "@angular-builders/custom-webpack:dev-server",
             "dependsOn": [
                 {
-                    "target": "prebuild",
+                    "target": "i18n",
```

Comment:
`prebuild` was renamed to `i18n` in https://github.com/Tinkoff/taiga-ui/pull/2567